### ISSUE TITLE
Upgrade Deis Workflow in Tokyo cluster

### ISSUE THREAD

**Issue by jgmize** [mozmeao/infra#446]
https://deis.com/docs/workflow/managing-workflow/upgrading-workflow/

**Comment by jgmize** [comment]
https://github.com/mozmeao/infra/blob/master/k8s/install/upgrade_workflow.sh

**Comment by jgmize** [comment]
After running `helm repo update` and then `./upgrade_workflow.sh v2.17.0` to try to upgrade to the [current release](https://github.com/deis/workflow/releases/tag/v2.17.0) I got the following error:

```
Customizing workflow
Removing monitor
Removing logger
Removing fluentd
Removing redis
Removing nsqd
Workflow customized
Upgrade lunging-eagle using v2.17.0? (ctrl-c to abort; set HELM_RELEASE_NAME env var to override)

Error: open v2.17.0: no such file or directory
```

**Comment by jgmize** [comment]
was giving it the wrong argument; the script requires the workflow values file and automatically uses the latest release. However, I'm now getting a new error:

```
$ ./upgrade_workflow.sh ~/mozilla/infra-private/k8s/clusters/tokyo/workflow_config_moz.yaml
Customizing workflow
Removing monitor
Removing logger
Removing fluentd
Removing redis
Removing nsqd
Workflow customized
Upgrade lunging-eagle using /home/jgmize/mozilla/infra-private/k8s/clusters/tokyo/workflow_config_moz.yaml? (ctrl-c to abort; set HELM_RELEASE_NAME env var to override)

Error: UPGRADE FAILED: render error in "workflow/charts/registry-proxy/templates/registry-proxy-daemon.yaml": template: workflow/charts/registry-proxy/templates/registry-proxy-daemon.yaml:11:25: executing "workflow/charts/registry-proxy/templates/registry-proxy-daemon.yaml" at <.Capabilities.KubeVe...>: can't evaluate field KubeVersion in type interface {}
```

**Comment by jgmize** [comment]
Resolved the issue above by installing the helm v2.6.0 and then running `helm init --upgrade` to upgrade tiller. Upgrade is proceeding now.
